src/testing-library/render.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { StoryFnMubanReturnType } from '@muban/storybook/dist/client/preview/types';
+import type { Story } from '@muban/storybook/dist/client/preview/types-6-0';
+import { createApp } from '@muban/test-mock';
+import type { BaseStoryFn } from '@storybook/addons/dist/ts3.9/types';
+import { getQueriesForElement, prettyDOM, queries } from '@testing-library/dom';
```

Comment:
Getting an eslint error here:
![image](https://user-images.githubusercontent.com/21334130/148750155-977c0b98-8385-45e9-8ecc-9649de77b6f9.png)

We should add `/src/testing-library` to `.eslintrc.js`'s  `import/no-extraneous-dependencies` rule `devDependencies` directory exclude. As this directory will only be used inside of `*.test` files.